src/main/java/com/mraof/minestuck/block/MSBlockShapes.java

```diff
@@ -100,6 +100,7 @@
 	public static CustomVoxelShape SMALL_ALCHEMITER = new CustomVoxelShape(new double[]{0,0,0,16,4,16}, new double[]{1,4,1,15,6,15}, new double[]{2,6,2,14,8,14}, new double[]{0.5,4,6.75,1,5,9.25}, new double[]{6.75,4,0.5,9.25,5,1}, new double[]{15,4,6.75,15.5,5,9.25}, new double[]{6.75,4,15,9.25,5,15.5}, new double[]{13.5,4,13.5,15.5,9,15.5}, new double[]{11.5,4,14.5,12.5,15,15.5}, new double[]{13,11,14.5,14,15,15.5}, new double[]{14.5,11,14.5,15.5,15,15.5}, new double[]{12,15,14.5,13.5,15.5,15.5}, new double[]{13.5,10.5,14.5,15,11,15.5}, new double[]{14,15,14,16,17,16});
 	
 	//Misc Machines
+	public static CustomVoxelShape INTELLIBEAM_LAZERSTATION = new CustomVoxelShape(new double[]{2,0,12,9,7,17}, new double[]{0,0,6,10,8,12}, new double[]{10,0,4,19,8,12}, new double[]{-1,7,7,20,10,11}, new double[]{4,7,11,9,18,15}, new double[]{2,18,9,12,24,15}, new double[]{-3,18,8,2,25,16});
```

Comment:
Looks like it's a bit larger than a block. Are you confident that it is within the size that vanilla can handle correctly for one block?

Reply:
Or actually, is the shape defined here even used for the lazerstation?

Reply:
it's not being used for anything at the moment because I wanted to put that off until I got the block most of the way finished, but after that I was thinking of changing the model to be 1, closer to the comic, and 2, better able to fit in minecraft